.npmrc

```diff
@@ -0,0 +1,12 @@
+public-hoist-pattern[]=@babel/core
+public-hoist-pattern[]=@babel/plugin-proposal-class-properties
+public-hoist-pattern[]=@babel/plugin-proposal-nullish-coalescing-operator
+public-hoist-pattern[]=@babel/plugin-proposal-optional-chaining
+public-hoist-pattern[]=@babel/plugin-transform-modules-commonjs
+public-hoist-pattern[]=@babel/preset-env
+public-hoist-pattern[]=@babel/preset-flow
+public-hoist-pattern[]=@babel/preset-react
+public-hoist-pattern[]=babel-plugin-minify-dead-code-elimination
+public-hoist-pattern[]=babel-plugin-transform-inline-environment-variables
+public-hoist-pattern[]=read-pkg-up
+public-hoist-pattern[]=resolve
```

Comment:
These are dependencies of the /dev/bable-preset which are required by the cli for transformations.

They need to be hoisted to the top level and made globally available due to the parcelBableRegister setting it's cwd to the project root

https://github.com/parcel-bundler/parcel/blob/alsh/rationalizing-phantom-dependencies/packages/dev/babel-register/index.js#L5